OpenSim/Analyses/InducedAccelerations.cpp

```diff
@@ -966,12 +966,14 @@ Array<bool> InducedAccelerations::applyContactConstraintAccordingToExternalForce
             if(exf->getPointExpressedInBodyName() != exf->getAppliedToBodyName()){
                 int appliedToBodyIndex = _model->getBodySet().getIndex(exf->getAppliedToBodyName());
                 if(appliedToBodyIndex < 0){
-                    cout << "External force appliedToBody " <<  exf->getAppliedToBodyName() << " not found." << endl;
+                    log_warn("External force appliedToBody '{}' not found.",
+                             exf->getAppliedToBodyName());
                 }
 
                 int expressedInBodyIndex = _model->getBodySet().getIndex(exf->getPointExpressedInBodyName());
                 if(expressedInBodyIndex < 0){
-                    cout << "External force expressedInBody " <<  exf->getPointExpressedInBodyName() << " not found." << endl;
+                    log_warn("External force expressedInBody '{}' not found.",
+                             exf->getPointExpressedInBodyName());
```

Comment:
For a user facing message do we need to use "expressedInBody"?

Reply:
I updated these messages to use the property names, which should be more meaningful to users.